auxiliary_tools/cdat_regression_testing/template_run_script.py

```diff
@@ -35,7 +29,5 @@
 # to the base results_dir, "/global/cfs/projectdirs/e3sm/e3sm_diags_cdat_test/".
 # Example: "671-lat-lon"
 SET_DIR = ""
-# TODO: Update SET_TO_NETCDF as needed.
-SAVE_NETCDF = True
 
-run_set(SET_NAME, SET_DIR, SAVE_NETCDF)
```

Comment:
I updated the template run script to always save `nc` files because a minority of the sets produce `.json` files.

We should focus on comparing `.nc` files anyways because it is a more accurate and useful comparison.